src/overlays/actors/ovl_En_Box/z_en_box.c

```diff
@@ -658,14 +658,14 @@ void EnBox_PostLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot,
 }
 
 Gfx* EnBox_SetRenderMode1(GraphicsContext* gfxCtx) {
-    Gfx* dl = GRAPH_ALLOC(gfxCtx, sizeof(Gfx) * 2);
+    Gfx* dl = GRAPH_ALLOC(gfxCtx, sizeof(Gfx));
```

Comment:
Could you change the ones in here to `gfx` and `head` to match elsewhere? Actually should probably make this consistent across the board.

Reply:
Went with `gfx` and `gfxHead`